src/js/views/maps/LayerDetailsView.js

```diff
@@ -210,21 +208,21 @@ define([
 
           // Select elements in the template that we will need to manipulate
           const sectionsContainer = this.el.querySelector(
-            "." + this.classes.sections,
+            `.${this.classes.sections}`,
           );
-          const labelEl = this.el.querySelector("." + this.classes.label);
+          const labelEl = this.el.querySelector(`.${this.classes.label}`);
 
           // Render each section in the Details panel
           this.renderedSections = _.clone(this.sections);
 
           // Remove and do not render opacity section if showOpacitySlider is false
-          if (model.get("showOpacitySlider") === false) {
+          if (model && model.get("showOpacitySlider") === false) {
```

Comment:
this is great, just FYI a shortcut for this is ` model?.get("showOpacitySlider")`

Reply:
Thank you @robyngit! I changed it